kahuna/app/views/main.scala.html

```diff
@@ -22,7 +22,7 @@
     <link rel="preconnect" href="@mediaApiUri"/>
     <link rel="preconnect" href="@authApiUri"/>
 
-    <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/grid-favicon-32.png")"/>
+    <link rel="shortcut icon" type="image/svg+xml" href="@routes.Assets.versioned("images/grid-favicon.svg")"/>
```

Comment:
What’s the browser support for Grid? Link icons SVG [isn’t supported everywhere](https://caniuse.com/link-icon-svg).

Reply:
I knew that and thought it’s fine. Apple continues to surprise me every single time. Thx, will amend. (and check if sizing changes haven’t broken “hinting”)

Reply:
Fixed.